src/view/screens/ProfileFeed.tsx

```diff
@@ -558,21 +543,20 @@ function AboutSection({
   feedRkey,
   feedInfo,
   headerHeight,
-  onScroll,
   scrollElRef,
   isOwner,
 }: {
   feedOwnerDid: string
   feedRkey: string
   feedInfo: FeedSourceFeedInfo
   headerHeight: number
-  onScroll: OnScrollHandler
   scrollElRef: React.MutableRefObject<ScrollView | null>
   isOwner: boolean
 }) {
   const pal = usePalette('default')
   const {_} = useLingui()
-  const scrollHandler = useAnimatedScrollHandler(onScroll)
+  const scrollHandlers = useScrollHandlers()
```

Comment:
Note: this is interesting because it shows how to get the contextual handlers when you're not using a `List`.

Reply:
Do you think we'll ever want to replace this `ScrollView` with an impl similar to `List`?

Reply:
So far it looks like almost all long scrolls we have are lists but could be done